app/interactors/create_temporary_stripe_credit_card.rb

```diff
@@ -25,10 +24,8 @@ def perform
     SchemaValidation.validate!(CardSchema::NewParams, credit_card_params)
 
     bank_account = if existing_bank_account = find_bank_account(@org, credit_card_params)
-                     # use this account
                      existing_bank_account
                    else
-                     # create new card in Stripe and add to our Stripe customer
                      create_stripe_card_bank_account(@org, stripe_tok, credit_card_params)
                    end
 
```

Comment:
This could better expressed as:

bank_account = find_bank_account(@org, credit_card_params) || create_stripe_card_bank_account(@org, stripe_tok, credit_card_params)